lib/views/hike_screen.dart

```diff
@@ -665,6 +684,36 @@ class _HikeScreenState extends State<HikeScreen> {
         false;
   }
 
+  LatLngBounds calculateMapBoundsFromListOfLatLng(List<LatLng> pointsList,
```

Comment:
It would be better to add this entire function in the Hike screen view model (which I guess isn't being used right now). So maybe create a new issue to refactor the Hike screen to make it more readable.

Reply:
Sure would do that.